crates/bdk/src/descriptor/mod.rs

```diff
@@ -600,7 +617,7 @@ mod test {
             "wpkh(02b4632d08485ff1df2db55b9dafd23347d1c47a457072a1e87be26896549a8737)",
         )
         .unwrap();
-        let psbt: psbt::PartiallySignedTransaction = deserialize(
+        let psbt = psbt::PartiallySignedTransaction::deserialize(
```

Comment:
```suggestion
        let psbt = psbt::Psbt::deserialize(
```

`bitcoin::psbt::Psbt` is available now. Next release hopefully `bitcoin::Psbt` will be (https://github.com/rust-bitcoin/rust-bitcoin/pull/1938)